apps/charterafrica/src/pages/index.page.js

```diff
@@ -586,6 +586,16 @@ export async function getStaticProps({ defaultLocale, locale, locales }) {
         siteDescription:
           "This website was created and maintained with the financial support of the European Union. Its contents are the sole responsibility of the European Partnership for Democracy, Africtivistes, Code for Africa, ECPDM, and Goree Institute and do not necessarily reflect the views of the European Union.",
       },
+      navbar: {
+        logo: {
+          alt: "Charter Africa",
+          src: "/images/charter-logo.svg",
+          width: "145",
+          height: "40",
```

Comment:
```suggestion
```
These need to be different for mobile vs desktop. Best to define them at respective components since we don't expect the logo to change, **_OR_** define size based on breakpoints and pick e.g.
```js
navbar: {
  logo: {
    // ...others,
    width: {
      xs: 145,
      md: 230,
    },
    // ...others
```
And in `DesktopNavBar`, you could do
```js
  <Logo {...logo} height={logo.height?.md || logo.height} width={logo.width?.md || logo.width} />
```
to pick appropriate size. Similarly for `MobileNavBar`.